qiskit_experiments/calibration_management/update_library.py

```diff
@@ -129,7 +129,7 @@ def update(
     @staticmethod
     def get_value(exp_data: ExperimentData, param_name: str, index: Optional[int] = -1) -> float:
         """A helper method to extract values from experiment data instances."""
-        candidates = exp_data.analysis_results(param_name)
+        candidates = exp_data.analysis_results(param_name, block=False)
```

Comment:
What is the aim of explicitly writing False here? To prevent deadlock because this is a part of analysis callback? Probably good to have some comment.

Reply:
Yes, if you don't do this it will hang indefinitely on the update cal.